ncp-web/js/ncp.js

```diff
@@ -268,21 +269,36 @@ $(function()
         var ret = $.parseJSON( result );
         if ( ret.token )
           $('#csrf-token').set( { value: ret.token } );
-        if ( ret.ret )                           // means that the process was launched
+        if ( "ret" in ret )                           // means that the process was launched
         {
-          if ( ret.ret == '0' ) 
+          if ( ret.ret == 0 )
           {
             if( ret.ref && ret.ref == 'nc-update' )
               window.location.reload( true );
             reload_sidebar();
             $('.circle-retstatus').set('+icon-green-circle');
           }
-          else 
+          else if ( ret.ret == -1 )
```

Comment:
```suggestion
          else if ( ret.ret == -1 )  // means we attached to an already running app
```

Reply:
Good point. Besides, I should probably use typed comparison here.

Reply:
Fixed in cd8451d318815945bae42ddda0a16b0a1ff1468a